runtime/onert/core/src/util/ShapeInference.cc

```diff
@@ -965,9 +965,14 @@ ir::Shape inferStridedSliceShape(const ir::Shape &input_shape, const StridedSlic
   return out_shape;
 }
 
-ir::Shape inferTileShape(const ir::Shape &in_shape, const int32_t *multiplier)
+ir::Shape inferTileShape(const ir::Shape &in_shape, const int32_t *multiplier,
+                         const int32_t multiplier_size)
 {
-  // assert(in_shape.rank() == multiplier.rank());
+  if (multiplier_size > in_shape.rank())
```

Comment:
I think the condition should be `if (multiplier_size == in_shape.rank())`
if multiplier_size < inshape.rank(), there may be array index overflow on line 980?

Reply:
It's my mistake. I will fix it.